src/App.css

```diff
@@ -1,38 +1,253 @@
+  margin: auto 30px auto 10px;
+  padding: 5px;
+  display: inline-block;
+}
+
+.wrapper {
+  display: flex;
+  justify-content: space-between;
+  width: calc(90%);
+  margin: 0 auto;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+ul.inventory {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  padding: 0;
+  margin: 50px auto;
+  width: calc(70%);
+  height: 100%;
+}
+
+li.inventory {
+  list-style: none;
+  padding: 1rem;
+  margin: 1.5rem 5px;
+  width: 160px;
+  height: 200px;
+  text-align: center;
   display: flex;
   flex-direction: column;
+  justify-content: space-around;
   align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  border: 1px solid #2EC4B6;
+  border-radius: 10px;
+}
+
+.add-to-cart {
+ width: 80%;
+ margin: 0 auto;
+ padding: 5px;
+ background-color: #FFBF69;
+ box-shadow: 1px 1px 10px #FF9F1C;
+ font-family: 'Halant', serif;
+ text-transform: uppercase;
+ letter-spacing: 1px;
+ border-radius: 10px;
+}
+
+p.inventory {
+  font-size: 1.2rem;
+}
+
+.product-name {
+  font-size: 1.3rem;
+  margin: 30px 0 0 0;
 }
 
-.App-link {
-  color: #61dafb;
+.price {
+  margin: 0;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
+p {
+  font-family: 'Halant', serif;
+}
+
+.cart {
+  width: calc(28%);
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  max-height: 80vh;
+  margin: 15px;
+}
+
+.cart h3 {
+  font-family: 'Iceberg', cursive;
+  width: 50%;
+  margin: 60px auto 0 auto;
+  border-bottom: 1px solid #FF9F1C;
+}
+
+.cart-container {
+  padding-left: 0;
+}
+
+.cart-list {
+  display: flex;
+  margin: 10px 0;
+  justify-content: space-around
+}
+
+.cart-list p {
+  text-align: left;
+  margin: auto 0;
+  display: inline-block;
+  font-size: 1.1rem;
+  width: 250px;
+}
+
+p.cart-price {
+  display: inline-block;
+  width: 100px;
+  text-align: end;
+  margin-right: 10px;
+}
+
+.empty {
+  text-transform: uppercase;
+}
+
+.remove {
+  background-color: #FFFFFF;
+  font-family: 'Halant', serif;
+  text-transform: uppercase;
+  letter-spacing: 1px;
+  font-size: 1rem;
+  border: none;
+  border-radius: 50%;
+  max-height: 26px;
+}
+
+@media(max-width: 1385px) {
+  li.inventory {
+    width: 140px;
+  }
+}
+
+@media(max-width: 1250px) {
+  li.inventory {
+    width: 130px;
+  }
+  .wrapper {
+    width: calc(95%)
+  }
+  .header-wrapper {
+    width: 95%;
+  }
+}
+
+@media(max-width: 1060px) {
+  .cart-list p {
+    font-size: 1rem;
+  }
+  ul.inventory {
+    margin: 50px 15px;
   }
-  to {
-    transform: rotate(360deg);
+  .cart {
+    width: calc(38%);
   }
 }
+
+@media(max-width: 925px) {
+  h1 {
+    font-size: 3rem;
+  }
+  .header h2 {
+    font-size: 1.2rem;
+  }
+  .cart-counter p {
+    font-size: 1.2rem;
+  }
+  .cart-counter img {
+    margin: auto 10px auto 10px;
+  }
+  li.inventory {
+    padding: 0.5rem;
+  }
+  p.product-name {
+    margin: 10px 0 0 0;
+  }
+}
+
+
```

Comment:
Very cleanly organized css code! Not much that I can see for improvement except if you wanted to write some comments discussing what you're doing, or grouping certain items together.